free/templatetags/free_extras.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
```

Comment:
This would be better to do using django custom context processors. I will adjust that.